community_supplied/Interface/snmp-interface-throughput.rule

```diff
@@ -1,11 +1,282 @@
+                * the input throughput utilization ($in-bw-util %) is greater than
+                * the threshold ($threshold %).
+                */
+                term is-input-bandwidth-abnormal {
+                    when {
+                        greater-than-or-equal-to "$in-bw-util" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name input throughput utilization ($in-bw-util %) crossed threshold value ($threshold %). Mbps:$in-mbps Bps:$input-bytes bps";
+                        }
+                    }
+                }
+                /*
+                * Defaults color to green.
+                */
+                term is-input-bandwidth-normal {
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name input throughput utilization ($in-bw-util %) is normal. Mbps:$in-mbps Bps:$input-bytes bps ";
+                        }
+                    }
+                }
+            }
+            trigger out-bw-utilization {
+                synopsis "Interface output traffic kpi";
+                description "Sets health based on output traffic exceed threshold";
+                frequency 180s;
+                /*
+                * Sets color to red and sends out an anomaly notification when
+                * the output throughput utilization ($out-bw-util %) is greater
+                * than the threshold ($threshold %).
+                */
+                term is-output-bandwidth-abnormal {
+                    when {
+                        greater-than-or-equal-to "$out-bw-util" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name output throughput utilization($out-bw-util%) crossed threshold value ($threshold %). Mbps:$out-mbps Bps:$output-bytes bps";
+                        }
+                    }
+                }
+                /*
+                * Defaults color to green.
+                */
+                term is-output-bandwidth-normal {
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name output throughput utilization ($out-bw-util %) is normal. Mbps:$out-mbps Bps:$output-bytes bps";
+                        }
+                    }
+                }
+            }
+            /*
+            * Variables with default values. Default values can be changed
+            * while deploying playbook instance.
+            */
+            variable interface-name-variable {
+                value .*;
+                description "Interfaces to be monitored";
```

Comment:
s/Interfaces to be monitored/Input interfaces to be monitored in regex/g

Reply:
Changed "Interfaces to be monitored" to "Enter interface name in regex i.e. ge-.*"